xarray/tests/test_ufuncs.py

```diff
@@ -172,6 +172,41 @@ def test_xarray_ufuncs_deprecation():
     with pytest.warns(PendingDeprecationWarning, match='xarray.ufuncs'):
         xu.cos(xr.DataArray([0, 1]))
 
+    with pytest.warns(None) as record:
+        xu.angle(xr.DataArray([0, 1]))
+    record = [el.message for el in record
+              if el.category == PendingDeprecationWarning]
+    assert len(record) == 0
+
+
+@requires_np113
+@pytest.mark.filterwarnings('ignore::RuntimeWarning')
+@pytest.mark.parametrize(
+    'name',
+    [name for name in dir(xu)
+        if (not name.startswith('_') and hasattr(np, name)
+            and name not in ['print_function', 'absolute_import', 'division'])]
```

Comment:
Here we list all ufuncs defined in `xarray.ufuncs`, but there are some functions that need to be filtered out as a result.